react/helpers/helpersTypesAccepted.spec.js

```diff
@@ -22,7 +22,7 @@ const makeMockFolder = () => ({
 describe('helpersTypesAccepted', () => {
   describe('getValidTypesAccepted', () => {
     it('should an array with all valid types', () => {
-      const res = getFileTypesAccepted('.pdf, text/plain, .not')
+      const res = getValidTypesAccepted('.pdf, text/plain, .not')
 
       expect(res).toStrictEqual(['.pdf', 'text/plain'])
     })
```

Comment:
**nit**: remarque non lié au code, mais au commit en général : par rapport à `the word "file" is not right because it can be a folder.` ça dépend de ce qu'on appelle `file`. Si c'est un doc du doctype `io.cozy.files` ça me choque pas qu'on utilise le terme `file` pour désigner un fichier ou un dossier, c'est à dire un doc de type file ou de type folder https://docs.cozy.io/en/cozy-doctypes/docs/io.cozy.files/ .

Item je trouve que ça n'apporte pas d'info sur le doctype... peut-être regarder ce qui se fait dans Drive et comment on nomme le doc venant du doctype ?

Reply:
- Je comprends, mais ici je trouvais plus simple d'être plus "basique" (file == fichier, folder == dossier)
- Avec la réflexion du premier point, je trouve ça compréhensible, en tout cas plus qu'anciennement `file` (qui dans ce contexte était vraiment trompeur je trouve)